pages/network/[asn].js

```diff
@@ -0,0 +1,86 @@
+import React, { useCallback, useMemo } from 'react'
+import { useRouter } from 'next/router'
+import { Container, Heading, Box, Flex } from 'ooni-components'
+import { FormattedMessage } from 'react-intl'
+import Form from 'components/network/Form'
+
+import Layout from 'components/Layout'
+import NavBar from 'components/NavBar'
+import { MetaTags } from 'components/dashboard/MetaTags'
+import FormattedMarkdown from 'components/FormattedMarkdown'
+
+import { MATContextProvider } from 'components/aggregation/mat/MATContext'
+
+import WebConnectivityChart from 'components/network/WebConnectivityChart'
+import MessagingAppsChart from 'components/network/MessagingAppsChart'
+import CircumventionToolChart from 'components/network/CircumventionToolsChart'
+
+const swrOptions = {
+  revalidateOnFocus: false,
+  dedupingInterval: 10 * 60 * 1000,
+}
+
+const fixedQuery = {
+  axis_x: 'measurement_start_day',
+}
+
+const testsOnPage = ['web_connectivity']
+const ChartsContainer = () => {
+  return (
+    testsOnPage.map(testName => (
```

Comment:
Why are is map being called on a list that has a single element? Isn't this equivalent to not having it being called at all?